ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/core_interfaces.schema.yml

```diff
@@ -9,3 +9,15 @@ keys:
   core_interfaces:
     type: dict
     $ref: "eos_designs#/$defs/l3_edge"
+    keys:
```

Comment:
I understand that you need this to print the correct description, but is messes the the docs too much, since inherited keys are placed below the rest. I suggest we go back to the more neutral "overrides `fabric_sflow` setting" again for these two. Apologies for the back and forth.

Reply:
Sure I have reverted this change, probably oppurtunity to change the behavior for overriden keys, and sort them based on original order itself.